src/runtime/internal/ssr.ts

```diff
@@ -32,16 +32,31 @@ export function spread(args, classes_to_add) {
 	return str;
 }
 
-export const escaped = {
+const ATTR_REGEX = /[&"]/g;
```

Comment:
I'm wondering based off a [comment from the original PR](https://github.com/ryansolid/dom-expressions/pull/27#issuecomment-690537305) if these should be moved instead `escape` since it looks they store state via the `lastIndex` attribute

Reply:
lastIndex is reset in the function